addons/base_vat/models/res_partner.py

```diff
@@ -155,7 +155,7 @@ def check_vat(self):
         for partner in self:
             country = partner.commercial_partner_id.country_id
             if partner.vat and self._run_vat_test(partner.vat, country, partner.is_company) is False:
-                partner_label = _("partner [%s]", partner.name)
+                partner_label = _("for partner [%s]", partner.name) if partner.name else ''
```

Comment:
please don't do that, don't split translations in serveral blocks, it's very hard to translate
https://www.odoo.com/documentation/15.0/developer/howtos/translations.html#blocks

Reply:
Ok indeed, I will instead just make a second return if not record_label. Thank you